master/docs/manual/configuration/report_generators/formatter.rst

```diff
@@ -44,17 +44,106 @@ The constructor of the class takes the following arguments:
     Use it only when necessary as this increases the overhead in terms of CPU and memory on the master.
 
 ``wantLogs``
-    This parameter (defaults to False) will extend the content of the steps of the given ``build`` object with the full logs of each steps from the build.
-    This requires ``wantSteps`` to be True.
+    Deprecated, use ``want_logs`` and ``want_logs_content`` set to the same value.
+
+``want_logs``
```

Comment:
shouldn't we also deprecate the want* in favor of want_*?

Reply:
I thought that this would be an unnecessary change. If I have a need to change wantSteps semantics before Buildbot 4.0 we can do that then without another deprecation.

Reply:
I think it would be best to at least try to be coherent on the parameter name within a single API. I think people can get that during transition, naming can be a bit incoherent, but within the same class, I feel it awkward.

Reply:
Fixed now.